docs/reports/reports.rst

```diff
@@ -3,4 +3,226 @@
+
+.. image:: images/mautic-report-data.png
+   :align: center
+   :alt: Screenshots of Mautic Report Data
+
+|
+
+* **Columns** - Select the columns of data that you want to appear in the table of data in the report. To select a column to appear, click the column name in the left column. To clear a column, click it from the right column. For example, if you select ID, it refers to the **ID** for the parent data source Category that you selected earlier (Contact ID, email ID, Asset ID, etc). It is recommended that you include **ID** in all reports.
+
+* **Order** - Select data Points to be used for sorting the data in the report. The available sort options are **Ascending** and **Descending**. To add multiple columns, click **Add Order**. For fields that use text, an **Ascending** order lists values starting with B after values starting with A (and so on). For number or date fields, the higher the number or later the date, the lower on the list the row is. Descending order is the opposite.
+
+.. note:: 
+
+   If you add multiple fields to order by, the last one in the Order list is used first. For example, if you order by **First Name Ascending** and then add **Email Ascending**, the Email column is sorted first and any duplicate rows is then sorted by first name ascending.
+
+* **Filters** - Filter the data by using conditions and values. Hence, you can generate granular reports. This option helps you to narrow down the data included in the report. The data points used for filters don’t have to be columns that appear in the report table. A commonly used filter for any reports that include Contact record data is Email Not Empty, which displays only identified Contacts in the Report. Additional use cases can be including Contacts or items that match a certain value, events happening within a certain date range, etc.
+
+.. note:: 
+
+   Setting the **Dynamic** option to **Yes** makes it easier for Users viewing the Report to change the data they see without actually editing the Report. Users can see the filter by opening the **Filters** drop-down from the top of the report page.
+
+* **Group by** - Select the columns for which you want to group data. By default, reports show all items individually. In many reports, you may see the same contact, company, or item appear multiple times. To only see each record listed once, you can add a grouping based on some attribute for the record. You can use Email or Contact ID to display a single row per Contact record. For example, you can group by **Contact ID** to view the unique number of Asset Downloads or Form Submissions for a single Form, instead of total Asset Downloads or Form Submissions, which could include duplicates.
```

Comment:
```suggestion
* **Group by** - Select the columns for which you want to group data. By default, Reports show all items individually. In many Reports, you may see the same Contact, Company, or item appear multiple times. To only see each record listed once, you can add a grouping based on some attribute for the record. 

You can use Email or Contact ID to display a single row per Contact record. For example, you can group by **Contact ID** to view the unique number of Asset Downloads or Form Submissions for a single Form, instead of total Asset Downloads or Form Submissions, which could include duplicates.
```